apps/web/src/pages/integrations/UpdateProviderPage.tsx

```diff
@@ -83,6 +84,24 @@ export function UpdateProviderPage() {
     },
   });
 
+  const { mutateAsync: deleteIntegrationApi, isLoading: isDeleting } = useMutation<
+    {},
+    { error: string; message: string; statusCode: number },
+    {
+      id: string;
+      name: string;
+    }
+  >(({ id }) => deleteIntegration(id), {
+    onSuccess: (_, vars) => {
+      successMessage(`Instance configuration ${vars.name} is deleted`);
+      refetch();
+      navigate('/integrations');
+    },
+    onError: (_, vars) => {
+      errorMessage(`Instance configuration ${vars.name} could not be deleted`);
```

Comment:
Could we add the error we get? To be more specific?

Reply:
The delete integration endpoint only returns the thrown error message and for me it is not helpful for the user. That is the reason why I made this generic message.